manager2/src/app/admin/projects/projects.component.html

```diff
@@ -35,8 +35,12 @@ <h3>Project creation</h3>
           </select>
         </div>
         <div class="col-sm-2">
-          <label for="project_size">Size (GB)</label>
-          <input placeholder="project size in G" type="number" id="project_size" [value]="default_size" [ngModelOptions]="{standalone: true}" [(ngModel)]="new_project.size" class="form-control"/>
+            <label for="project_size">Size (GB)</label>
+            <input placeholder="project size in GB" type="number" id="project_size" [value]="default_size" [ngModelOptions]="{standalone: true}" [(ngModel)]="new_project.size" class="form-control"/>
+        </div>
+        <div class="col-sm-2">
+            <label for="project_cpu">Cpu (Hour)</label>
+            <input placeholder="project cpu in G" type="number" id="project_cpu" [value]="default_cpu" [ngModelOptions]="{standalone: true}" [(ngModel)]="new_project.cpu" class="form-control"/>
```

Comment:
In G?

Reply:
fixed: 5ecb403008f78cc3716119dfae809234dba93363